src/views/HomePage.vue

```diff
@@ -1,35 +1,24 @@
-<script>
-import { defineComponent } from 'vue';
-
-import { mapState } from 'pinia';
+<script setup lang="ts">
+import { computed } from 'vue';
 
 import HeroSection from '@/components/HomePage/HeroSection/HeroSection.vue';
 import MoviesSection from '@/components/HomePage/MoviesSection/MoviesSection.vue';
 import ContactDetails from '@/components/ContactPage/ContactDetails.vue';
+import ScreeningsPage from '@/views/ScreeningsPage.vue';
 
 import { useMoviesStore } from '@/store/movies';
+const moviesStore = useMoviesStore();
 
-export default defineComponent({
-  name: 'HomePage',
-  components: {
-    HeroSection,
-    MoviesSection,
-    ContactDetails
-  },
-  computed: {
-    ...mapState(useMoviesStore, ['allMovies']),
-    comingSoon() {
-      return this.allMovies.filter((movie, index) => index < 3);
-    }
-  }
+const comingSoon = computed(() => {
+  return moviesStore.allMovies.filter((movie, index) => index < 3);
```

Comment:
As a good practice if you are not using the value (so that your IDE/linter won't yell at you) you can omit it by naming it `_`
```suggestion
  return moviesStore.allMovies.filter((_, index) => index < 3);
```

Reply:
Thanks for the tip!
Maybe my linter is too lax, I've not heard anything about this from it.. 🤔